components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -132,95 +141,125 @@ export default function ProjectSettingsView() {
                     Update Name
                 </Button>
             </form>
-            <Heading2 className="mt-12">Prebuilds</Heading2>
-            <Subheading>Choose the workspace machine type for your prebuilds.</Subheading>
-            <div className="max-w-md">
-                <SelectWorkspaceClassComponent
-                    selectedWorkspaceClass={project.settings?.workspaceClasses?.prebuild}
-                    onSelectionChange={setWorkspaceClassForPrebuild}
+            <div className="">
+                <Heading2 className="mt-12">Prebuilds</Heading2>
+                <CheckboxInputField
+                    label="Enable prebuilds"
+                    hint="You need to have admin permissions on the repository in order to install a webhook for prebuilds."
+                    checked={enablePrebuilds}
+                    onChange={(checked) => updateProjectSettings({ enablePrebuilds: checked })}
                 />
-            </div>
-            <CheckboxInputField
-                label="Enable Incremental Prebuilds"
-                hint={
-                    <span>
-                        When possible, use an earlier successful prebuild as a base to create new prebuilds. This can
-                        make your prebuilds significantly faster, especially if they normally take longer than 10
-                        minutes.{" "}
-                        <a className="gp-link" href="https://www.gitpod.io/changelog/faster-incremental-prebuilds">
-                            Learn more
-                        </a>
-                    </span>
-                }
-                checked={project.settings?.useIncrementalPrebuilds ?? false}
-                onChange={(checked) => updateProjectSettings({ useIncrementalPrebuilds: checked })}
-            />
-            <CheckboxInputField
-                label="Cancel Prebuilds on Outdated Commits"
-                hint="Cancel pending or running prebuilds on the same branch when new commits are pushed."
-                checked={!project.settings?.keepOutdatedPrebuildsRunning}
-                onChange={(checked) => updateProjectSettings({ keepOutdatedPrebuildsRunning: !checked })}
-            />
-            <CheckboxInputField
-                label={
-                    <span>
-                        Use Last Successful Prebuild{" "}
-                        <PillLabel type="warn" className="font-semibold mt-2 ml-2 py-0.5 px-2 self-center">
-                            Alpha
-                        </PillLabel>
-                    </span>
-                }
-                hint="Skip waiting for prebuilds in progress and use the last successful prebuild from previous
+                <div className="flex mt-4 max-w-2xl">
+                    <div className="flex flex-col ml-6">
```

Comment:
Would be great to use `<InputField>` to handle wrapping the `<SelectWorkspaceClassComponent>` (see [here](https://github.com/gitpod-io/gitpod/blob/5c84263c6f027bf3349fb476cf4c23d392db3dbc/components/dashboard/src/user-settings/Account.tsx#L163-L165) for an example) so it stays consistent with other input fields/labels.

```jsx
<InputFieldComponent label="Workspace machine type">
  <SelectWorkspaceClassComponent... />
</InputFieldComponent>
```